internal/errs/userfacing.go

```diff
@@ -1,10 +1,14 @@
 package errs
 
+import "errors"
+
 type UserFacingError interface {
 	error
 	UserError() string
 }
 
+type ErrOpts func(err *userFacingError) *userFacingError
```

Comment:
```suggestion
type ErrOpt func(err *userFacingError) *userFacingError
```

The type is only covering a single option.